### PR TITLE
added copy to clipboard 

### LINKED ISSUE #21: Add share to social links
It would be super helpful if users could share the links to social media apps in a click.

### PR DESCRIPTION
**Pull request by ProCode2**
Added copy to clipboard as a part of social links area, resource cards.

Closes #21

### CODE REVIEW

**Comment by ProCode2** [comment]
Some tailwind classes not working in production.

File: components/HelplineCard.jsx [hunk 1]
```diff
@@ -21,6 +19,27 @@ const HelplineCard = ({
 }) => {
     return (
         <div className="w-full bg-white rounded-lg shadow dark:bg-gray-1200 dark:text-gray-300">
+            <div className="w-full">
```
Comment (Deep-Codes) [inline components/HelplineCard.jsx:22]: Compare this :

https://life-5xhbeu4bk-coronasafe.vercel.app/delhi_(nct)/central_delhi/helpline

with

https://life.coronasafe.network/delhi_(nct)/central_delhi/helpline

Screenshot 2021-04-24 at 11 22 24 PM

Screenshot 2021-04-24 at 11 22 11 PM

Card's Layout break

File: components/AmbulanceCard.jsx [hunk 2]
```diff
@@ -18,6 +19,27 @@ const AmbulanceCard = ({
...
+                                marginLeft: 'auto'
+                            }}
+                            onClick={() => {
+                                copyToClipboard(`
+                                Name: ${name ? name : 'Ambulance'}
+                                Contact: ${phone1}
+                                `);
+                                alert('Copied!');
+                            }}>
+                            <FontAwesomeIcon
```
Comment (Deep-Codes) [inline components/AmbulanceCard.jsx:32]: add `cursor: pointer` for cursor

File: components/AmbulanceCard.jsx [hunk 3]
```diff
@@ -18,6 +19,27 @@ const AmbulanceCard = ({
         <>
             <div className="w-full">
                 <div className="bg-white rounded-lg shadow dark:bg-gray-1200 dark:text-gray-300">
+                    <div className="w-full">
+                        <p
```
Comment (Deep-Codes) [inline components/AmbulanceCard.jsx:23]: Screenshot 2021-04-24 at 11 26 34 PM

Seems to be breaking...

File: components/HelplineCard.jsx (hunk 1 above)
Comment (ProCode2) [reply components/HelplineCard.jsx:22]: The code that's in the main is the same as what's in mine that affects that layout.

Comment (ProCode2) [reply components/HelplineCard.jsx:22]: The cards have become unmanageable.

File: components/AmbulanceCard.jsx (hunk 3 above)
Comment (shrihari689) [reply components/AmbulanceCard.jsx]: Fixed 📍

File: components/AmbulanceCard.jsx (hunk 2 above)
Comment (shrihari689) [reply components/AmbulanceCard.jsx:32]: Fixed 📍

File: components/HelplineCard.jsx (hunk 1 above)
Comment (shrihari689) [reply components/HelplineCard.jsx]: Fixed 📍

**Review by Deep-Codes — APPROVED** [review]

**Pull request merged**